src/Lhm/Chunker.php

```diff
@@ -86,13 +90,25 @@ public function __construct(AdapterInterface $adapter, \Phinx\Db\Table $origin,
         $this->sqlHelper = $sqlHelper ?: new SqlHelper($this->adapter);
 
         $this->options = $options + [
-                'stride' => 2000,
-                'strideMax' => 20000,
-                'targetQuerySeconds' => 0.75
-            ];
+            'stride' => 2000,
+            'strideMax' => 20000,
+            'targetQuerySeconds' => 0.75,
+        ];
 
         $this->primaryKey = $this->adapter->quoteColumnName($this->sqlHelper->extractPrimaryKey($this->origin));
 
+        $adapterFactory = AdapterFactory::instance();
+        $res = $this->adapter->query('SELECT HOST FROM INFORMATION_SCHEMA.PROCESSLIST WHERE COMMAND = \'Binlog Dump\'')->fetchAll();
```

Comment:
Allow overriding this via a config in addition to this fetchAll

Reply:
oh nuts.. forgot about that bit